### PR TITLE
Update 115.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/115.com.xml [hunk 1]
```diff
@@ -1,10 +1,21 @@
 <!--
+	Issues after login:
+		Only http by server:
+			- ^115.com
```
Comment (J0WI) [inline src/chrome/content/rules/115.com.xml]: HTTPS works, but there is a JS redirect back to HTTP

File: src/chrome/content/rules/115.com.xml [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+		Use Mismatch sources https://static.115cdn.com/static/pc/d_155.html
+			- home.115.com
+		Refresh home page continued:
+			- msg.115.com
+			- q.115.com
```
Comment (J0WI) [inline src/chrome/content/rules/115.com.xml]: I could not confirm this, can you give examples?

File: src/chrome/content/rules/115.com.xml (hunk 1 above)
Comment (ivysrono) [inline src/chrome/content/rules/115.com.xml]: Which JS?

File: src/chrome/content/rules/115.com.xml (hunk 2 above)
Comment (ivysrono) [inline src/chrome/content/rules/115.com.xml]: Could you login?

Comment (J0WI) [inline src/chrome/content/rules/115.com.xml]: I'm not logged in

Comment (ivysrono) [inline src/chrome/content/rules/115.com.xml]: 18911129518
3Ykgvx3K

**Comment by J0WI** [comment]
It seems that they changed the login procedure, so I can no longer reproduce the issues.

**Pull request merged**